flow/scripts/synth.tcl

```diff
@@ -67,8 +67,16 @@ if {[info exist ::env(BLOCKS)]} {
 synth  -top $::env(DESIGN_NAME) {*}$::env(SYNTH_ARGS)
 
 if { [info exists ::env(USE_LSORACLE)] } {
+    set lso_script [open $::env(OBJECTS_DIR)/lso.script w]
+    puts $lso_script "ps -a"
+    puts $lso_script "oracle --config $::env(LSORACLE_KAHYPAR_CONFIG)"
+    puts $lso_script "ps -m"
+    puts $lso_script "crit_path_stats"
+    puts $lso_script "ntk_stats"
+    close $lso_script
```

Comment:
Running this on GCD I get the following error from Yosys:
```
Number of cells:                521
     $lut                          382
     BUF_X16                         2
     BUF_X32                         2
     BUF_X8                          2
     DFF_X1                         35
     INV_X16                         2
     NAND2_X1                       34
     NAND2_X2                        2
     NAND2_X4                       60
```
It's importing as $lut, which do not get techmapped.

Reply:
I've added an explicit techmap step after the lsoracle call, in case the techmapping on lines 85-101 is not run due to the conditionals.

Reply:
that does work, but based on the plugin code, it looks like this should not be needed and it can map directly to the right cells. So maybe something isn't getting setup correctly to cause it to use the builtin library to re-integration.

Reply:
MAJ3 isn't available in the built-in library, so they're treated as LUTs, necessitating a call to techmap. Adding MAJ as a Yosys primitive is on our wishlist to avoid the extra step.  In the interim, manually catching other types of cell is useful internally if we're running in interactive mode, using custom scripts, etc. It's not very useful in this context.

Reply:
Ahh I see, well the techmap step appears to remap the luts, so I guess that will do until yosys is updated.